core/src/main/scala/kafka/tools/ProducerPerformance.scala

```diff
@@ -38,7 +38,12 @@ import org.apache.log4j.Logger
 object ProducerPerformance extends Logging {
 
   def main(args: Array[String]) {
+    run(args)
+    System.exit(0)
+  }
 
+  def run(args: Array[String]) {
+    println("Starting run"+args.deep.mkString)
```

Comment:
Nit: there is usually a space before and after the `+` in the Kafka codebase (there are some exceptions though).